pulsar-broker-auth-oidc/src/main/java/org/apache/pulsar/broker/authentication/model/OpenIDProviderMetadata.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.authentication.model;
```

Comment:
There's not much value in putting models in a different package. Some developers do that, but that is considered an anti-pattern. If you ask me, "package by feature" is the recommended approach. 
That would mean to put this class together with the other classes where this class is used.

Reply:
Moved to a package by feature paradigm.